### PR TITLE
More CI

### PR DESCRIPTION
**Pull request by CohenCyril**
- fourcolor 8.11 and 8.12
- odd-order 8.11 and 8.12
- bigenough 8.11 and 8.12
- finmap    8.11 and 8.12

##### Things done/to do

- ~added corresponding entries in `CHANGELOG_UNRELEASED.md`~
- ~added corresponding documentation in the headers~

##### Automatic note to reviewers

Read [this Checklist](https://github.com/math-comp/math-comp/wiki/Checklist-for-following,-reviewing-and-playing-with-a-PR#checklist-for-reviewing-a-pr) and make sure there is a milestone.

### CODE REVIEW

**Comment by pi8027** [comment]
How about fcsl-pcm?

**Comment by anton-trunov** [comment]
FWIW, coq-fcsl-pcm is fully compatible with Coq 8.12

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -390,6 +430,16 @@ ci-fcsl-pcm-8.10:
...
 
+ci-fcsl-pcm-8.11:
+  extends: .ci-fcsl-pcm
+  variables:
+    COQ_VERSION: "8.11"
+
+ci-fcsl-pcm-8.10:
+  extends: .ci-fcsl-pcm
+  variables:
+    COQ_VERSION: "8.11"
```
Comment (pi8027) [inline .gitlab-ci.yml:441]:
```suggestion
ci-fcsl-pcm-8.12:
  extends: .ci-fcsl-pcm
  variables:
    COQ_VERSION: "8.12"
```

**Comment by pi8027** [comment]
I hope the issue has been disappeared thanks to math-comp/finmap#71. @CohenCyril Could you rebase to refresh CI?

**Comment by pi8027** [comment]
Since there is no response here, let me commit my above suggestion to see if it works.

**Pull request merged**